core/assets/bundles/bundle.properties

```diff
@@ -198,12 +198,12 @@ server.outdated = [crimson]Outdated Server![]
 server.outdated.client = [crimson]Outdated Client![]
 server.version = [gray]v{0} {1}
 server.custombuild = [accent]Custom Build
-confirmban = Are you sure you want to ban this player?
-confirmkick = Are you sure you want to kick this player?
-confirmvotekick = Are you sure you want to vote-kick this player?
+confirmban = Are you sure you want to ban "{0}[white]"?[]
```

Comment:
These three lines also have redundant tags. `[]` at the end doesn't do anything.